ibis/formats/tests/test_polars.py

```diff
@@ -60,6 +70,25 @@ def test_to_from_ibis_type(ibis_dtype, polars_type):
     assert PolarsType.to_ibis(polars_type, nullable=False) == ibis_dtype(nullable=False)
 
 
+@pytest.mark.parametrize(
+    ("ibis_dtype", "polars_type"),
+    [
+        param(
+            dt.Timestamp(scale=0), pl.Datetime("ns", time_zone=None), id="timestamp_sc0"
+        ),
+        param(dt.Timestamp(), pl.Datetime("ns", time_zone=None), id="timestamp_scn"),
+        param(
+            dt.Timestamp("UTC", scale=0),
+            pl.Datetime("ns", time_zone="UTC"),
+            id="timestamp_tz_sc0",
+        ),
+    ],
+)
+def test_from_ibis_type_seconds(ibis_dtype, polars_type):
+    # we accept seconds as an ibis type, default to ns in polars
```

Comment:
@jcrist in the original issue, you suggested maybe defaulting to `"ms"` for when we go from `"s"` in Ibis to polars. I though it made sense  keeping the behavior we had before for that particular case which is `"ns"`. Any objections?

Reply:
It still seems a bit odd to me that asking for seconds I would get back milliseconds. I say we raise in the seconds case as an `UnsupportedOperationError` the same any other missing functionality.

Reply:
This will break the current behavior where we ask for seconds and return `ns`. Just making sure we are ok with breaking that.